src/components/Nav/NavBottomContainer/NavBottomContainer.scss

```
@@ -0,0 +1,40 @@
+@import '../../../styles/variables.scss';
+
+.navBottomContainer {
+  @include colors();
+  .navbar {
+    @include flex(space-between, center);
+    padding: 1px 50px 1px 50px;
```

margin, padding에 속성을 부여할 경우 상황에 따라서 shorthand property를 이용해서 손쉽게 부여할 수 있습니다.
```suggestion
    padding: 1px 50px;
```